how-to-guides/switch-data-training.ipynb

```diff
@@ -0,0 +1,106 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# How to switch data provider during training"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "User can easily switch data provider during the training using\n",
+    "[`set_data()`](https://pytorch.org/ignite/v0.4.5/generated/ignite.engine.engine.Engine.html#ignite.engine.engine.Engine.set_data). See an example in the\n",
+    "documentation of the method."
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "source": [
+    "from ignite.engine import Engine, Events\n",
+    "\n",
+    "data1 = [1, 2, 3]\n",
+    "data2 = [4, 5, 6]\n",
```

Comment:
To make the logging more clear that we changed the data previously I used 
```
data2 = [10 + a, 10 +b, 10 + c]
```
such we clearly see the differences between batches